frontend/src/lib/usePlan.ts

```diff
@@ -1,9 +1,36 @@
+import type { SetStateAction } from "jotai";
 import { useAtom } from "jotai";
 
 import { type ExtendedCourse, planFamily } from "@/atoms/planFamily";
 
 import type { Registration } from "./types";
 
+export interface PlanState {
```

Comment:
Nie chcemy wszystkich interfejsów razem trzymać w types?

Reply:
chcemy, lint bartusia nie chce

Reply:
jak nie pozwala

Reply:
@Rei-x 
![image](https://github.com/user-attachments/assets/17fee7c3-a8ac-492a-807c-818c63b8980f)

Reply:
ale Marcinowi chodziło to żeby wrzucić to do folderu types 😭

Reply:
aaaaaa